src/TesterInternal/General/BuiltInSerializerTests.cs

```diff
@@ -29,7 +29,7 @@ public class BuiltInSerializerTests
         public void InitializeForTesting()
         {
             TraceLogger.Initialize(new NodeConfiguration());
-            SerializationManager.Initialize(false, null);
+            SerializationManager.Initialize(false, null, false);
```

Comment:
So this runs with binary fallback ser. 
Thus 2 questions:
1) why the changes above in the test w.r.t. Parse instead of Cast. 
2) Do you want to add some new tests that will actualy execute json fallback ser?